spec/controllers/projects_controller_spec.rb

```diff
@@ -305,25 +306,90 @@
     end
   end
 
-  describe "#copy" do
+  describe "#copy_form" do
     let(:project) { create(:project, identifier: "blog") }
 
-    it "renders 'copy'" do
-      get "copy", params: { id: project.id }
-      expect(response).to be_successful
-      expect(response).to render_template "copy"
+    shared_examples_for "successful request" do
+      it "renders 'copy_form'", :aggregate_faulures do
```

Comment:
typo: `:aggregate_faulures` => `aggregate_failures`

```suggestion
      it "renders 'copy_form'", :aggregate_failures do
```

Reply:
good catch! thanks